src/lang/std/engineConnection.ts

```diff
@@ -1176,6 +1182,94 @@ export class EngineCommandManager {
       }
     })
   }
+  private async initPlanes() {
```

Comment:
Oh I thought you were going to add all of this to kclManager? did I remember that wrong or did you change your mind?

Reply:
yeah I was going to but then I kept hitting "enginemanager cannot be started before initialized" or some weird error but now that im saying it out loud it might have jsut been the hot reload in dev mode and not a real issue, anyways we can work thru this in our meeting today!